syntax = "proto3";


message HandleTypeProto {
    int32 handle    = 1;
}
